File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/varargs_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'varargs'

if Varargs.test("Hello") != "Hello"
  raise RuntimeError, "Failed"
end

f = Varargs::Foo.new("Greetings")
if f.str != "Greetings"
  raise RuntimeError, "Failed"
end

if f.test("Hello") != "Hello"
  raise RuntimeError, "Failed"
end

File: support_tools/swigwin-3.0.12/Examples/test-suite/csharp/varargs_runme.cs
// varargs test

using System;
using varargsNamespace;

public class varargs_runme {

  public static void Main() {

    if (varargs.test("Hello") != "Hello")
      throw new Exception("Failed");

    Foo f = new Foo("Greetings");
    if (f.str != "Greetings")
      throw new Exception("Failed");
        
    if (f.test("Hello") != "Hello")
      throw new Exception("Failed");
  }
}
